app/models/widget.rb

```diff
@@ -0,0 +1,16 @@
+class Widget < ActiveRecord::Base
+  enum widget_type: Constants::WIDGET_TYPES
```

Comment:
Oh my bad, I remember now, actually meant to do it without the freeze, but the _RuboCop_ mislead me again... I spend some time researching whether to use text or `enum`, and come to the conclusion that using `text` (or `jsonb` perhaps) would give us no additional benefits (or in other words, problems that could occur with `enum` could very well occur with `text`).